src/DrDotnet.Profilers/api/metadata_import.rs

```diff
@@ -141,4 +143,132 @@ impl MetadataImportTrait for MetadataImport {
             _ => Err(hr),
         }
     }
+
+    fn get_nested_class_props(&self, td: crate::ffi::mdTypeDef) -> Result<crate::ffi::mdTypeDef, HRESULT> {
+        let mut enclosing_type_def = MaybeUninit::uninit();
+        let hr = unsafe {
+            self.import().GetNestedClassProps(
+                td,
+                enclosing_type_def.as_mut_ptr()
+            )
+        };
+
+        match hr {
+            HRESULT::S_OK => {
+                let enclosing_type_def = unsafe { enclosing_type_def.assume_init() };
+                Ok(enclosing_type_def)
+            }
+            _ => Err(hr),
+        }
+    }
+
+    fn enum_generic_params(&self, td: crate::ffi::mdTypeDef) -> Result<Vec<crate::ffi::mdGenericParam>, HRESULT> {
+        let mut enumerator_handle = MaybeUninit::uninit();
+        let mut generic_param_buffer = Vec::<mdGenericParam>::with_capacity(10);
+        unsafe { generic_param_buffer.set_len(10) };
+        let mut params_fetched = MaybeUninit::uninit();
+
+        info!("open 2");
+
+        let hr = unsafe {
+            self.import().EnumGenericParams(
```

Comment:
Not sure why but it seems that crashes. I have checked the arguments and the vtable and it all seems correct... `IMetadataImport2::GetVersionName` works and is further in the vtable so I think everything is correctly aligned